### PR TITLE
[ISSUES #174]optimize quick start

### PR DESCRIPTION
**Pull request by odbozhou**
## What is the purpose of the change

XXXXX

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily. Notice, `it would be helpful if you could finish the following 5 checklist(the last one is not necessary)before request the community to review your PR`.

- [x] Make sure there is a [Github issue](https://github.com/apache/rocketmq-connect/issues) filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [x] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [x] Write necessary unit-test(over 80% coverage) to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/rocketmq/tree/master/test).
- [x] Run `mvn -B clean apache-rat:check findbugs:findbugs checkstyle:checkstyle` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -119,49 +75,94 @@ curl -X POST -H "Content-Type: application/json" http://127.0.0.1:8082/connector
...
-## 8.启动sink connector
+## 5.启动sink connector
 
 ```
-curl -X POST -H "Content-Type: application/json" http://127.0.0.1:8082/connectors/fileSinkConnector -d '{"connector-class":"org.apache.rocketmq.connect.file.FileSinkConnector","filename":"~/test/test-sink-file.txt","connect-topicname":"fileTopic"}'
+curl -X POST -H "Content-Type: application/json" http://127.0.0.1:8082/connectors/fileSinkConnector -d '{"connector-class":"org.apache.rocketmq.connect.file.FileSinkConnector","filename":"test-sink-file.txt","connect-topicname":"fileTopic"}'
+
+cat test-sink-file.txt
 ```  
 看到一下日志说明file sink connector启动成功了
```
Comment (RongtongJin) [inline README.md:87]: 看到一下日志 -> 看到以下日志

File: README.md [hunk 2]
```diff
@@ -119,49 +75,94 @@ curl -X POST -H "Content-Type: application/json" http://127.0.0.1:8082/connector
...
 ```
     GET请求  
     http://(your worker ip):(port)/connectors/(connector name)/stop
+    
+    停止demo中的两个connector
+    curl     http://127.0.0.1:8082/connectors/fileSinkConnector/stop
+    curl     http://127.0.0.1:8082/connectors/fileSourceConnector/stop
+    
 ```  
 看到一下日志说明connector停止成功了
```
Comment (RongtongJin) [inline README.md:117]: 同上

File: README.md [hunk 3]
```diff
@@ -1,114 +1,70 @@
...
+查看日志文件
 
- ${user.home}/logs/rocketmqconnect 
+tail -100f ~/logs/rocketmqconnect/connect_runtime.log
 
-## 6.配置文件
+以下日志表示runtime启动成功：
 
-持久化配置文件默认目录 ～/storeRoot
+The standalone worker boot success.
```
Comment (duhenglucky) [inline README.md:48]: 需要在最后面介绍standalone 跟distributed 的区别，以及增加 distribute 的部署方式

File: README.md (hunk 1 above)
Comment (odbozhou) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (odbozhou) [reply README.md]: done

File: README.md [hunk 4]
```diff
@@ -1,114 +1,70 @@
...
+
+echo "Hello \r\nRocketMQ\r\n Connect" > test-source-file.txt
+
+curl -X POST -H "Content-Type: application/json" http://127.0.0.1:8082/connectors/fileSourceConnector -d '{"connector-class":"org.apache.rocketmq.connect.file.FileSourceConnector","filename":"test-source-file.txt","connect-topicname":"fileTopic"}'
 ```
 
-   看到一下日志说明file source connector启动成功了
-   
-   2019-07-16 11:18:39 INFO pool-7-thread-1 - Source task start, config:{"properties":{"source-record-...
+看到一下日志说明file source connector启动成功了
```
[the remark this answers is not in the document]
Comment (odbozhou) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (odbozhou) [reply README.md:51]: quick start可以先不解释，先快速跑起来。具体区别可以放到connect 介绍或者101文档里介绍。

**Review by duhenglucky — APPROVED** [review]

**Pull request merged**